superset/models/dashboard.py

```diff
@@ -218,7 +220,7 @@ def filter_sets_lst(self) -> Dict[int, FilterSet]:
         }
 
     @property
-    def charts(self) -> List[BaseDatasource]:
+    def charts(self) -> List[str]:
```

Comment:
How/why wasn’t this caught by Mypy?

Reply:
It was caught when I updated the file! This must be a very old piece of code.